rasa_core/evaluate.py

```diff
@@ -463,16 +444,15 @@ def collect_story_predictions(
     report, precision, f1, accuracy = get_evaluation_metrics(
             [1] * len(completed_trackers), correct_dialogues)
 
-    in_training_data_fraction = \
-        _in_training_data_fraction(is_in_training_data_list)
+    in_training_data_fraction = _in_training_data_fraction(action_list)
 
     log_evaluation_table([1] * len(completed_trackers),
                          "END-TO-END" if use_e2e else "CONVERSATION",
                          report, precision, f1, accuracy,
                          in_training_data_fraction,
                          include_report=False)
 
-    return story_eval_store, failed, in_training_data_fraction
+    return story_eval_store, failed, action_list, in_training_data_fraction
```

Comment:
might make sense to make a namedtuple out of this? I think for items is quite a lot